src/codeu/chat/client/ClientUser.java

```diff
@@ -102,6 +102,20 @@ public void addUser(String name) {
     }
   }
 
+  public void addNickname(String name) {
+    boolean validInputs = isValidName(name);
+
+    User user = (validInputs) ? controller.newNickname(current.id, name) : null;
+
+    if (user == null) {
+      LOG.info("Error: nickname not created - %s.\n",(validInputs) ? "server failure" : "bad input value");
```

Comment:
please split this line - too long
i.e. 
LOG.info("stuff",
    (validInputs) ? ...);
note: +4 space indent for split lines
also, space after comma